package/linux-firmware/linux-firmware.mk

```diff
@@ -537,11 +537,13 @@ LINUX_FIRMWARE_IWL8000_UCODE_API_MAX = 36
 LINUX_FIRMWARE_IWL8265_UCODE_API_MAX = 36
 LINUX_FIRMWARE_IWL9000_UCODE_API_MAX = 46
 LINUX_FIRMWARE_IWL_22000_UCODE_API_MAX = 77
-LINUX_FIRMWARE_AX210_UCODE_API_MAX = 83
+# Capped by linux-firmware, Linux 6.12 has max 89
+LINUX_FIRMWARE_IWL_AX210_UCODE_API_MAX = 86
 # The 9560 driver has maximum ucode API defined in ax210.c yet its versions
 # are seemingly tracking the 22000 series.
-LINUX_FIRMWARE_AX210_UCODE_API_MAX_9560 = 77
-LINUX_FIRMWARE_IWL_BZ_UCODE_API_MAX = 83
+LINUX_FIRMWARE_IWL_AX210_UCODE_API_MAX_9560 = 77
+# Capped by linux-firmware, Linux 6.12 has max 93
+LINUX_FIRMWARE_IWL_BZ_UCODE_API_MAX = 86
```

Comment:
Hm, actually, the kernel also says `IWL_BZ_UCODE_API_MIN 90` for this one :thinking: 

I guess not much we can do other than bumping linux-firmware which is a bigger thing. I just wonder if that means that the move to 6.12 will break such devices in practice? :thinking:

Reply:
> Hm, actually, the kernel also says `IWL_BZ_UCODE_API_MIN 90` for this one 🤔

Oh, good catch! :facepalm: 

> I guess not much we can do other than bumping linux-firmware which is a bigger thing. 

We can't (easily) move to the latest one, as the size of new wildcard firmwares (esp. `amdgpu`) bloated the image to the point where it exceeds rootfs partition size. Version 90 was introduced in `20240513`, I'll see how's the bloat there.

> I just wonder if that means that the move to 6.12 will break such devices in practice? 🤔

I'm pretty sure it will - the FW loader will try to load firmware version decrementing down to the minimum supported one, and if it doesn't find any file, the card will stay unprovisioned.